### ISSUE TITLE
gulp run-yo does not work on Windows due to pathing issues

### ISSUE THREAD

**Issue by MrPiao** [OfficeDev/generator-office#49]
The problem seems to be right here in gulpfile.js:

```javascript
  spawn('node',
    [
      '--debug',
      path.join(which.sync('yo'), '../../', 'lib/node_modules/yo/lib/cli.js'),
      'office',
      ' --skip-install'
    ], {
      stdio: 'inherit'
    });
```

This ends up looking for cli.js in the wrong place:

```
C:\Users\alexpark\OneDrive\fun\generator-office>gulp run-yo
[21:34:11] Using gulpfile ~\OneDrive\fun\generator-office\gulpfile.js
[21:34:11] Starting 'run-yo'...
[21:34:11] BE AWARE!!! - Running this with default options will scaffold the project in the generator's source folder.
[21:34:11] Finished 'run-yo' after 15 ms
Debugger listening on port 5858
module.js:338
    throw err;
          ^
Error: Cannot find module 'C:\Users\alexpark\AppData\Roaming\lib\node_modules\yo\lib\cli.js'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```

Changing ` path.join(which.sync('yo'), '../../', 'lib/node_modules/yo/lib/cli.js'),` to ` path.join(which.sync('yo'), '../../', 'npm/node_modules/yo/lib/cli.js'),` works, but a proper fix would switch based on the OS or maybe programmatically find the npm path?

**Comment by andrewconnell** [comment]
The change won't work for people who do not install NPM global modules in the default location for a stock node install. This is actually quite common to avoid the admin permission prompt when installing addins globally.

**Comment by andrewconnell** [comment]
Merged into **dev** in PR #68
